packages/simulator/web/routes.js

```diff
@@ -125,10 +133,10 @@ function register(io) {
           capacity,
         })
       ),
-      throttleTime(50)
+      bufferCount(10)
```

Comment:
Add parameters to send remaining buffer 

const throttleLossless = (time, count) => (stream) => stream.pipe(
   windowTime(100), // start a window every x ms
      mergeMap((win) =>
        win.pipe(
          groupBy((car) => car.id), // create a stream for each car in this window
          mergeMap((cars) => cars.pipe(last())) // take the last update in this window
        )
      ),
)